Tests/New_Customer/test_new_customer.py

```diff
@@ -20,6 +21,17 @@ def test_new_customer_input_message(self):
 
         newCustomer = NewCustomerPage(self.driver)
         newCustomer.click_on_input("Customer Name")
+        time.sleep(2)
+        newCustomer.click_on_input("City")
+        time.sleep(2)
+        newCustomer.click_on_input("State")
+        time.sleep(2)
+        newCustomer.click_on_input("PIN")
+        time.sleep(2)
+        newCustomer.click_on_input("Phone_number")
+        time.sleep(2)
+        newCustomer.click_on_input("Email")
+        time.sleep(2)
```

Comment:
Using the `*` operator is a good idea. I think this case may cause some trouble when it comes to the click-on input step. Instead, I think we could do this:
```suggestion
        time.sleep(2)
        newCustomer.click_on_input(input_name)
```